libs-private/db-management/indexGenerator.ts

```diff
@@ -105,12 +105,8 @@ const collectionsWithIndexes: Record<string, IndexSpecification[]> = {
     { field: { deleted: 1 }, name: 'deleted_1' },
     { field: { environment: 1 }, name: 'environment_1' },
     { field: { createdAt: 1 }, name: 'createdAt_1' },
-    { field: { state: 1 }, name: 'state_and_createdAt_1' },
-    { field: { createdAt: 1 }, name: 'state_and_createdAt_1' },

```

Comment:
Why did we delete these ones?

Reply:
They are added in the combined object below!